LICENSES.third-party

```diff
@@ -146,6 +146,27 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 
+jquery.graphviz.svg (https://github.com/mountainstorm/jquery.graphviz.svg/)
```

Comment:
For consistency with the above/below, should the title/prelude to the license inclusion etc say what its for and where it is in the code base?

Reply:
Good idea. Added an explanation and notes on the path where the JS is placed.